cypress/integration/owm.spec.js

```diff
@@ -4,17 +4,22 @@ describe('OpenWeatherMap layer', function() {
     cy.wait(300)
     cy.window().then((win) => {
       cy.get('.leaflet-tile-pane').children().should('have.length', 1)
-      cy.get('#map-city label').click({ force: true })
+      cy.get('#map-Clouds label').click({ force: true })
```

Comment:
Nice - will this be more stable? Like, there are clouds everywhere even though there aren't cities everywhere?

Reply:
Yeah clouds,temp and pressure seemed to be more general to me among other maps. I just went with clouds.